packages/editor/src/components/editors/third-party/interface.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/jsx-key */
+import { IvyIcons } from '@axonivy/ui-icons';
+import type { ElementType } from '@axonivy/inscription-protocol';
+import type { ReactNode } from 'react';
+import { memo } from 'react';
+import InscriptionEditor from '../InscriptionEditor';
+import { useGeneralPart, useProgramInterfaceStartPart, useConfigurationPart } from '../../../components/parts';
+
+const RuleInterfaceEditor = memo(() => {
```

Comment:
ThirdPartyProgramInterfaceEditor

there can also be other thirdparty program interface activities...